uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -469,6 +469,22 @@ define([
       }
 
       /**
+       * Flattens statuses from object to array of objects. This is used to
+       * populate the dropdown with array of statuses.
+       *
+       * @return {Array}
+       */
+      this.getStatuses = function () {
+        if (this.requestStatuses) {
```

Comment:
`requestStatuses` should be initialized right at the beginning of the controller, so you can be sure it's always defined